interface/forms/observation/new.php

```diff
@@ -14,265 +14,37 @@
  */
 
 require_once(__DIR__ . "/../../globals.php");
+/**
+ * @global string $srcdir defined in globals.php
+ */
+global $srcdir;
 require_once("$srcdir/api.inc.php");
 require_once("$srcdir/patient.inc.php");
 require_once("$srcdir/options.inc.php");
 require_once($GLOBALS['fileroot'] . '/custom/code_types.inc.php');
 
-use OpenEMR\Common\Csrf\CsrfUtils;
-use OpenEMR\Common\Forms\ReasonStatusCodes;
-use OpenEMR\Core\Header;
-
-$returnurl = 'encounter_top.php';
-$formid = (int)(isset($_GET['id']) ? $_GET['id'] : 0);
-
-if ($formid) {
-    $sql = "SELECT * FROM `form_observation` WHERE id=? AND pid = ? AND encounter = ?";
-    $res = sqlStatement($sql, array($formid, $_SESSION["pid"], $_SESSION["encounter"]));
-
-    $all = [];
-    for ($iter = 0; $row = sqlFetchArray($res); $iter++) {
-        $all[$iter] = $row;
-    }
-    $check_res = $all;
+use Symfony\Component\HttpFoundation\Request;
+use OpenEMR\Services\ObservationService;
+use OpenEMR\Controllers\Interface\Forms\Observation\ObservationController;
+use OpenEMR\Common\Logging\SystemLogger;
+
+$logger = new SystemLogger();
+
+// Output the response
+try {
+    // Create controller and handle request
+    $request = Request::createFromGlobals();
+    $service = new ObservationService();
+    $controller = new ObservationController($service);
+    $response = $controller->new($request);
+    $response->send();
+    exit;
```

Comment:
These exits are unconditional, too. If they can't be removed, there could be one after the block.